packages/fxa-payments-server/src/routes/Subscriptions/index.test.tsx

```diff
@@ -53,6 +54,13 @@ import {
 import FlowEvent from '../../lib/flow-event';
 jest.mock('../../lib/flow-event');
 
+jest.mock('./PaymentUpdateFormV2', () => ({
```

Comment:
Since `PaymentUpdateFormV2.test.tsx` hosts all the unit tests for the new form, we can just skip them here and mock the component for verifying the feature flip logic.